src/hooks/useSessionState.ts

```diff
@@ -9,9 +9,9 @@ export type TSessionState = {
 
 export const useSessionState = (): TSessionState => {
   const { user } = useAuth0();
-  const [sessionState, setSessionState] = React.useState({});
+  const [sessionState, setSessionState] = useState<any>({});
```

Comment:
<img width="598" alt="Screen Shot 2020-10-31 at 3 24 20 PM" src="https://user-images.githubusercontent.com/37982292/97790314-38126380-1b8d-11eb-8179-7474ec6206f2.png">

Unsure why this is throwing an error, it seems like on the first few render passes, the return value from useState is the value itself, not the `[state, setState]` signature.